src/components/Nav/Nav.scss

```
@@ -0,0 +1,53 @@
+@import '/src/styles/mixin.scss';
+@import '/src/styles/variables.scss';
+
+.nav {
+  @include flexSort(space-between, center);
+  background-color: white;
+  padding: 0px 15%;
+  height: 80px;
+  @include initBorderStyle(bottom);
+
+  .home {
+    @include flexCenter;
+    .imgHome {
+      @include btnForIcon($large);
+      margin: 20px;
+    }
```

```suggestion
  .home {
    @include flexCenter;
    
    .imgHome {
      @include btnForIcon($large);
      margin: 20px;
    }
```
코드 가독성을 위해 스타일 끼리의 간격을 띄워주세요!

---

넵! 알겠습니다

---

수정 완료했습니다.